core/common/assets/js/api/core/commands.js

```diff
@@ -325,13 +462,54 @@ export default class Commands extends CommandsBackwardsCompatibility {
 	 * Method fired after the command runs.
 	 *
 	 * @param {string} command
+	 * @param {{}} args
+	 * @param {*} results
 	 */
-	afterRun( command ) {
-		const component = this.getComponent( command ),
-			container = component.getRootContainer();
+	afterRun( command, args, results = undefined ) {
```

Comment:
Try rename (we already have runAfter)

Reply:
Try now https://github.com/elementor/elementor/pull/14973/commits/73c379b00ba81f1804d35b4707346869982ebe41